middleware/cache/heap.go

```
@@ -41,7 +41,7 @@
 }
 
 func (h *indexedHeap) Push(x any) {
-	h.pushInternal(x.(heapEntry)) //nolint:forcetypeassert // Forced type assertion required to implement the heap.Interface interface
+	h.pushInternal(x.(heapEntry)) //nolint:forcetypeassert,errcheck // Forced type assertion required to implement the heap.Interface interface
```

golangci-lint

directive `//nolint:forcetypeassert,errcheck // Forced type assertion required to implement the heap.Interface interface` is unused for linter "errcheck"

(nolintlint)